packages/devtools_app/lib/src/screens/deep_link_validation/validation_details_view.dart

```diff
@@ -0,0 +1,309 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app_shared/ui.dart';
+import 'package:flutter/material.dart';
+
+import '../../shared/common_widgets.dart';
+import '../../shared/table/table.dart';
+import '../../shared/ui/colors.dart';
+import 'deep_link_list_view.dart';
+import 'deep_links_controller.dart';
+import 'deep_links_model.dart';
+
+class ValidationDetailView extends StatelessWidget {
+  const ValidationDetailView({
+    super.key,
+    required this.linkData,
+    required this.viewType,
+    required this.controller,
+  });
+
+  final LinkData linkData;
+  final TableViewType viewType;
+  final DeepLinksController controller;
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
this build method has gotten pretty large. Please break up into helper widgets to make this more readable.